### DISCUSSION TITLE
Notification History

### DISCUSSION THREAD

**Discussion by Looodon** [runelite/runelite#17929] [Ideas]
Hey
It would help a lot to have a notification's history section for two reasons:
1) Keep track of certain events that you might have missed if you AFK, switch world, re-launch client the next day, cleared chat
2) To find problem plugins that are firing notifications even when logged out while the client is active

In the case of # 2, currently I'm facing an issue where when I'm logged out and the client is minimized, but I'm still receiving notifications. There is no way for me to find out because Runelite prints the notifs to the in-game chat so this would help me find out which one

**Comment by Nightfirecat** [comment]
I would expect notification history to be something your operating system can provide for you.

**Reply by Looodon** [reply]
I'm on windows 11, I did Windows logo key + N to launch Notifications Centers and it's empty if that's what you mean
